internal/dogrun/adapters/repository/dogrun_repository.go

```diff
@@ -31,7 +32,7 @@ PlaceIDで、ドッグランの取得
 func (drr *dogrunRepository) GetDogrunByPlaceID(c echo.Context, placeID string) (model.Dogrun, error) {
 	logger := log.GetLogger(c).Sugar()
 	dogrun := model.Dogrun{}
-	if err := drr.db.Preload("DogrunTags.TagMst").Preload("RegularBusinessHours").Preload("SpecialBusinessHours").Where("place_id = ?", placeID).Find(&dogrun).Error; err != nil {
+	if err := drr.db.Preload("DogrunTags").Preload("RegularBusinessHours").Preload("SpecialBusinessHours").Where("place_id = ?", placeID).Find(&dogrun).Error; err != nil {
```

Comment:
みたいに改行した方が見やすいな
このままsuggest許可しないでフォーマッターしてほしい
```suggestion
	if err := drr.db.Preload("DogrunTags").
	Preload("RegularBusinessHours").
	Preload("SpecialBusinessHours").
	Where("place_id = ?", placeID).
	Find(&dogrun).Error; err != nil {
```